### DISCUSSION TITLE
Quick thought

### DISCUSSION THREAD

**Discussion by AntoineRestivo** [benoitseron/BosonSampling.jl#48] [Q&A]
Do we really need to import CSV and DataFrames inside BosonSampling? Usually you generate datasets outside the package right? I mean, it won't be super relevant to provide the script that generates the data. I rather call those two libraries in the file (that I won't make public) that generates the data. What do you think? 
Also, perhaps it's even better if we stick to only one type of data, JLD no? And I would rather use CSV for benchmarks

**Comment by benoitseron** [comment]
I need them for my own part of the code - basically all the packages are
used by default.

Benoît Seron.
